ts/features/zendesk/components/__tests__/ZendeskSupportComponent.test.tsx

```diff
@@ -62,7 +70,29 @@ describe("the ZendeskSupportComponent", () => {
   });
 
   describe("when the user press the zendesk open ticket button", () => {
-    it.todo("should call openTicket");
+    it("if panic mode is false, should navigate to the ZendeskAskPermissions screen", () => {
+      const navigateToZendeskAskPermissionsSpy = jest.spyOn(
+        zendeskNavigation,
+        "navigateToZendeskAskPermissions"
+      );
+      const store = createStore(appReducer, globalState as any);
+      const component = renderComponent(store);
+      const zendeskButton = component.getByTestId("contactSupportButton");
+      fireEvent(zendeskButton, "onPress");
+      expect(navigateToZendeskAskPermissionsSpy).toBeCalled();
+    });
+    it("if panic mode is true, should navigate to the ZendeskAskPermissions screen", () => {
+      const navigateToZendeskPanicModeSpy = jest.spyOn(
+        zendeskNavigation,
+        "navigateToZendeskPanicMode"
+      );
+      const store = createStore(appReducer, globalState as any);
+      const component = renderComponent(store);
+      store.dispatch(getZendeskConfig.success(mockZendeskConfig));
+      const zendeskButton = component.getByTestId("contactSupportButton");
+      fireEvent(zendeskButton, "onPress");
+      expect(navigateToZendeskPanicModeSpy).toBeCalled();
```

Comment:
Tests look great 🎉 !
Since they test mutually-exclusive behaviour, I suggest to add an assertion about the other function _not_ being called:

```typescript
     // in this test
      expect(navigateToZendeskPanicModeSpy).toBeCalled();
      expect(navigateToZendeskAskPermissionsSpy).not.toBeCalled();
```

And the other way round at line [82](https://github.com/pagopa/io-app/pull/3674/files#diff-afd4ce25055a8b1d15873525bef205dacdb815013d065f360f09f41eb91f2fb3R82).

Reply:
Thank you for the suggestion.
Added in [0d33bd6](https://github.com/pagopa/io-app/pull/3674/commits/0d33bd6e86b04bfe1c5b3012cda011b4bd58b825)